### ISSUE TITLE
Border width bugs

### ISSUE THREAD

**Issue by Ferm86** [enolgor/wow-addon-tiptac#21]
Hey :)

First of all thanks for keeping this addon alive. I'm not a native english speaker but I will try my best to describe the issue.

I set up the border width of my tooltips to 1px solid. Most of them are displayed correctly. But some tend to thin out one side of the tooltip by 1px. That means in my setup, the border isn't shown. If I set up the border to 1.5px or 2px, one side of the tooltip is thinner. I put up a couple of screenshots to show you what I mean.

![tooltips](https://user-images.githubusercontent.com/73543966/97323219-d9897480-1870-11eb-950c-db1127ba062f.png)

Thanks again for your effort!

**Comment by enolgor** [comment]
I can't reproduce it... could you send a screenshot with your relevant configuration?

**Comment by Ferm86** [comment]
Yeah sure. I don't know exactly what you need. So I put out the ones I think could be relevant. If you need something else, please let me know :)

![settings](https://user-images.githubusercontent.com/73543966/97752151-c33f1b00-1af3-11eb-9b29-a4b37508891d.png)

P.S.: I just installed the latest release. The issue still exist for me.
